README-PSR16.md

```diff
@@ -0,0 +1,33 @@
+# Momento PSR-16 Client Library
+
+The Momento PSR-16 client library, `Psr16SImpleCache`, implements the [PHP PSR-16 common interface for caching
```

Comment:
I think your capital `I` should be lower case here?  That really through me off for a sec, I was like "what's an `ImpleCache`?"